src/main/java/app/bpartners/geojobs/service/event/readme/ReadmeLogCreatedService.java

```diff
@@ -0,0 +1,67 @@
+package app.bpartners.geojobs.service.event.readme;
+
+import static java.net.http.HttpClient.Version;
+import static java.net.http.HttpClient.newHttpClient;
+import static java.net.http.HttpRequest.BodyPublishers;
+import static java.net.http.HttpResponse.BodyHandlers;
+
+import app.bpartners.geojobs.endpoint.event.model.readme.ReadmeLogCreated;
+import app.bpartners.geojobs.endpoint.rest.readme.monitor.ReadmeMonitorConf;
+import app.bpartners.geojobs.model.exception.BadRequestException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.util.Base64;
+import java.util.List;
+import java.util.function.Consumer;
+import lombok.RequiredArgsConstructor;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class ReadmeLogCreatedService implements Consumer<ReadmeLogCreated> {
+  private static final String README_AUTH_PREFIX = "Basic ";
+  private static final String README_API_MIME_TYPE = "application/json";
+  private final ObjectMapper objectMapper;
+  private static final HttpClient httpClient = newHttpClient();
+
+  @Override
+  @SneakyThrows
+  public void accept(ReadmeLogCreated readmeLogCreated) {
+    var readmeMonitorConf = readmeLogCreated.getReadmeMonitorConf();
+    var readmeLog = readmeLogCreated.getReadmeLog();
+
+    if (readmeMonitorConf.isDevelopment() != readmeLog.development()) {
+      throw new BadRequestException(
+          "readmeLog.development should be " + readmeMonitorConf.isDevelopment());
+    }
+
+    String requestBody = objectMapper.writeValueAsString(List.of(readmeLog));
```

Comment:
Put it into another method, then handle exception with try/catch and remove `@SneakyThrows`